v03_pipeline/lib/tasks/exports/write_new_entries_parquet.py

```diff
@@ -0,0 +1,145 @@
+    WriteRemappedAndSubsettedCallsetTask,
+)
+
+ANNOTATIONS_TABLE_TASK = 'annotations_table_task'
+HIGH_AF_VARIANTS_TABLE_TASK = 'high_af_variants_table_task'
+REMAPPED_AND_SUBSETTED_CALLSET_TASKS = 'remapped_and_subsetted_callset_tasks'
+
+
+@luigi.util.inherits(BaseLoadingRunParams)
+class WriteNewEntriesParquetTask(BaseWriteParquetTask):
+    def output(self) -> luigi.Target:
+        return GCSorLocalTarget(
+            new_entries_parquet_path(
+                self.reference_genome,
+                self.dataset_type,
+                self.run_id,
+            ),
+        )
+
+    def requires(self) -> list[luigi.Task]:
+        return {
+            ANNOTATIONS_TABLE_TASK: (
+                self.clone(UpdateVariantAnnotationsTableWithNewSamplesTask)
+            ),
+            REMAPPED_AND_SUBSETTED_CALLSET_TASKS: [
+                self.clone(
+                    WriteRemappedAndSubsettedCallsetTask,
+                    project_i=i,
+                )
+                for i in range(len(self.project_guids))
+            ],
+            **(
+                {
+                    HIGH_AF_VARIANTS_TABLE_TASK: self.clone(
+                        UpdatedReferenceDatasetQueryTask,
+                        reference_dataset_query=ReferenceDatasetQuery.high_af_variants,
+                    ),
+                }
+                if ReferenceDatasetQuery.high_af_variants
+                in BaseReferenceDataset.for_reference_genome_dataset_type(
+                    self.reference_genome,
+                    self.dataset_type,
+                )
+                else {}
+            ),
+        }
+
+    def create_table(self) -> None:
+        unioned_ht = None
+        for project_guid, remapped_and_subsetted_callset_task in zip(
+            self.project_guids,
+            self.input()[REMAPPED_AND_SUBSETTED_CALLSET_TASKS],
+            strict=True,
+        ):
+            mt = hl.read_matrix_table(remapped_and_subsetted_callset_task.path)
+            ht = compute_callset_family_entries_ht(
+                self.dataset_type,
+                mt,
+                get_fields(
+                    mt,
+                    self.dataset_type.genotype_entry_annotation_fns,
+                    **self.param_kwargs,
+                ),
+            )
+            ht = ht.annotate(
+                family_entries=hl.enumerate(ht.family_entries).starmap(
+                    lambda i, fs: hl.enumerate(fs).starmap(
+                        lambda j, e: e.annotate(
+                            family_guid=ht.family_guids[i],  # noqa: B023
+                            sampleId=ht.family_samples[ht.family_guids[i]][j],  # noqa: B023
+                        ),
+                    ),
+                ),
+            )
+            annotations_ht = hl.read_table(
+                self.input()[ANNOTATIONS_TABLE_TASK].path,
+            )
+            ht = ht.join(annotations_ht)
+
+            if self.input().get(HIGH_AF_VARIANTS_TABLE_TASK):
```

Comment:
I feel like we will want this column for GRCh37 as well, given that in the clickhosue backend this column is designed to reduce the search space in a proband-only family, which will be the same order of magnitude across genome builds. But we can test that out and build it if necessary when we get to that part of the implementation